src/pages/Home/Home.css

```
@@ -0,0 +1 @@
+/* TODO: Add CSS specific to component. */
```

```suggestion
// TODO: Add styling.
```

Nit. You're using the multiline syntax for a single line comment. 

This file is already in same directory as the component, I don't think you need to mentioned the CSS will be specific to the component. Same as your other files.